klaus/highlighting.py

```diff
@@ -34,17 +36,18 @@ def _format_lines(self, tokensource):
 
     def _lookup_ctag(self, token):
         matches = list(self._get_all_ctags_matches(token))
-        best_matches = self.get_best_ctags_matches(matches)
-        if not best_matches:
+        try:
+            best_match = next(self.get_best_ctags_matches(matches))
```

Comment:
I just looked at the length of the lists on an example page and the overhead is unnoticeable, so I guess I was overzealous indeed :)